SDBlockDevice.cpp

```diff
@@ -594,7 +602,7 @@ bd_size_t SDBlockDevice::get_program_size() const
 
 bd_size_t SDBlockDevice::get_erase_size() const
 {
-    return _erase_size;
+    return _block_size;
```

Comment:
Can we leave a comment, stating erase size is used for erase operations of flash devices in FAT filesystem and should be equivalent to block size, as no erase is required for 
sd cards before every write operation.

Reply:
I would actually propose we remove the erase function from this driver completely. See https://github.com/ARMmbed/mbed-os/pull/4983. Updating...

Reply:
@geky - Work in progress, will remove erase function and add trim instead